client/news/525.feature

```diff
@@ -0,0 +1 @@
+Use server side operations for modifying indexes if the devpi-server version supports it. This prevents undoing changes if two clients modify the same index concurrently.
```

Comment:
Change "undoing" to something like "last one wins and overwrites previous change"